frontend/src/components/Features/Skeleton/ProductSkeleton.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Skeleton from 'react-loading-skeleton';
+
+function ProductSkeleton(props) {
+  return (
+    <div className="product">
+      <figure className="product__media">
+        <Skeleton height={'100%'} width={'100%'} />
```

Comment:
```suggestion
        <Skeleton height={100} width={100} />
```
Thử ngắn gọn hơn như này xem nhé

Reply:
Không được anh ạ, em đang muốn để width và height là 100%. Nếu truyền width, height là 100 thì nó hiểu là 100px